### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by wxy1234567** [megvii-research/HDR-Transformer#2]
Hi, can you share the pretrained model? The model I trained according to the default parameters is not very effective, and some test images still have obvious ghosts. I guess it may be caused by the unadjusted learning rate, optimizer and other parameters?

**Comment by liuzhen03** [comment]
Hi, we are now retraining the model on Kalantari17 ’s and Prabhakar19’s datasets as the released version has been reorganized. The pretrained model will be released after we finish the double-check. Feel free to reopen this issue if you have any questions.
